npc/warps/fields/abyss_warper.txt

```diff
@@ -39,43 +39,43 @@
 
 hu_fild05,168,304,0	script	Column#abyss1	HIDDEN_NPC,{
 
-	mes "^3355FFThere are 3 oddly";
-	mes "shaped grooves on";
-	mes "the surface of this";
-	mes "column. It seems that";
-	mes "certain dragon body parts";
-	mes "would fit perfectly into";
-	mes "the column grooves.^000000";
+	mes("^3355FFThere are 3 oddly");
```

Comment:
as this is single sentence split into multiple lines, I recommend that you use \r
Example:

```
mes("^3355FFThere are 3 oddly\r"
	"shaped grooves on\r"
	"the surface of this\r"
	"column. It seems that\r"
	"certain dragon body parts\r"
	"would fit perfectly into\r"
	"the column grooves.^000000";
```